### PR TITLE
data: aarpardev

### PR DESCRIPTION
**Pull request by aarpardev**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7255"></a>

### CODE REVIEW

File: data/aarpardev.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Aaron Parsons",
+  "displayStatsPublic": false,
```
Comment (khushalarora) [inline data/aarpardev.json:3]: You can remove this line, it is not required anymore :+1: 
```Suggestion
```

File: data/aarpardev.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "type": "personal",
+  "bio": "Full-Stack Software Developer | React | Node.js | JavaScript",
+  "avatar": "https://avatars.githubusercontent.com/u/115501471?s=96&v=4",
```
Comment (khushalarora) [inline data/aarpardev.json:6]: You can also remove this line, it is not required anymore :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @aarpardev, thank you for contributing :tada:

**Comment by aarpardev** [comment]
Hi, khushalarora!

Thank you for the suggested updates. I commited the changes in my branch and will look over the complete LinkFree documentation further before I make any further updates.

**Review by Sindhuinti — DISMISSED** [review]
Looks good 🎉

File: data/aarpardev.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Aaron Parsons",
+  "type": "personal",
```
Comment (SaraJaoude) [inline data/aarpardev.json:3]: due to a recent update this is no longer required:
```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**